app/src/main/java/ch/ubique/notifyme/app/network/KeyLoadWorker.java

```diff
@@ -19,7 +19,7 @@
 
 public class KeyLoadWorker extends Worker {
 
-	public static final String NEW_NOTIFICATION = "NEW_NOTIFICATION";
+	public static final String NEW_NOTIFICATION_BROADCAST = "NEW_NOTIFICATION_BROADCAST";
```

Comment:
Intent actions values should be canonical, i.e. prefixed with the package name. And maybe named like `ACTION_NEW_NOTIFICATION`?

Reply:
Same for the other action keys in the app.

Reply:
Yes, thanks. I have changed this throughout the app.

Reply:
There is a `.` missing between the package anem and the value.